server/src/server.ts

```diff
@@ -61,6 +61,10 @@ const diagSeverityMap = new Map<string, DiagnosticSeverity>([
 
 /** Regex that matches paths containing the `.generated-yml` directory. */
 const GENERATED_YML_DIR_REGEX = /(^|\/)\.generated-yml\//;
+/** Regex that matches the `_FILE_TYPE_ F` instruction. */
+const FILE_TYPE_F = /^_FILE_TYPE_\s+F\s*/;
+/** Regex that matches the `_FILE_TYPE_ M` instruction. */
+const FILE_TYPE_M = /^_FILE_TYPE_\s+M\s*/;
```

Comment:
Consider (for both of these regexps) replacing the final `\s*` with a `\b`. It expresses what we’re actually looking for slightly better. Here you have some sort of weird optional-ish fake lookahead xD